doc/reference/power_management/index.rst

```diff
@@ -124,6 +124,27 @@ have higher wake latencies. Following is a thorough list of available states:
 .. doxygenenumvalue:: PM_STATE_SOFT_OFF
    :project: Zephyr
 
+.. _pm_constraints:
+
+Power States Constraint
+=======================
+
+The power management subsystem allows different Zephyr components and
+applications to set constraints on various power states preventing the
+system to go these states. This can be used by devices when executing
+tasks in background to avoid the system to go to state where it would
+lose context. Constraints can be set, released and checked using the
+follow APIs:
+
+.. doxygenfunction:: pm_constraint_set
+   :project: Zephyr
+
+.. doxygenfunction:: pm_constraint_release
+   :project: Zephyr
+
+.. doxygenfunction:: pm_constraint_get
+   :project: Zephyr
+
```

Comment:
Additionally, I think it could be useful to add a mention about `device_busy_foo` apis here, to explain the difference from  power manager point of view.

Reply:
We need to update device power management documentation indeed, but right now I'm just looking system power management, is that ok ?

Reply:
Sure